File: packages/graph/__fixtures__/extractor-project/AuthTest.java
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Nested;

class AuthTest {

    @Test
    @DisplayName("should reject expired tokens")
    void rejectExpiredTokens() {
        assert true;
    }

    @Test
    @DisplayName("should accept valid JWT tokens")
    void acceptValidJwt() {
        assert true;
    }

    @Nested
    @DisplayName("Login Flow")
    class LoginFlow {
        @Test
        @DisplayName("should require email and password")
        void requireCredentials() {
            assert true;
        }

        @Test
        @DisplayName("should lock account after 5 failed attempts")
        void lockAfterFailedAttempts() {
            assert true;
        }
    }

    @Test
    void testPasswordHashing() {
        assert true;
    }
}

File: packages/graph/__fixtures__/extractor-project/auth.test.ts
import { describe, it, test, expect } from 'vitest';

describe('AuthService', () => {
  describe('token validation', () => {
    it('should reject expired tokens', () => {
      expect(true).toBe(true);
    });

    it('should accept valid JWT tokens', () => {
      expect(true).toBe(true);
    });

    test('handles malformed token gracefully', () => {
      expect(true).toBe(true);
    });
  });

  describe('login', () => {
    it('should require email and password', () => {
      expect(true).toBe(true);
    });

    it('should lock account after 5 failed attempts', () => {
      expect(true).toBe(true);
    });
  });
});
